src/System.Net.Sockets/tests/FunctionalTests/UnixDomainSocketTest.netcoreapp.cs

```diff
@@ -221,8 +212,10 @@ public void Socket_SendReceive_Success()
                     {
                         int bytesRead;
                         byte[] buffer = new byte[readBufferSize];
-                        while ((bytesRead = await client.ReceiveAsync(new ArraySegment<byte>(buffer), SocketFlags.None)) > 0)
+                        while (readData.Length < writeBuffer.Length)
                         {
+                            bytesRead = await client.ReceiveAsync(new ArraySegment<byte>(buffer), SocketFlags.None);
+                            Assert.InRange(bytesRead, 1, writeBuffer.Length - readData.Length);
```

Comment:
I changed this because in my build there are some issues with shutdown, and the test would sometimes hang.

Reply:
@stephentoub - For the hang, as discussed offline, couple of similar bugs were fixed in the release branch. If you can give it a try on a recent build.

Reply:
> If you can give it a try on a recent build.

I will, thanks.